src/Settings/settings.json

```diff
@@ -737,5 +737,14 @@
         "desc": "Open the old file when parsing an old problem URL",
         "type": "bool",
         "tip": "If a problem URL was set for a file path, when parsing that problem\nfrom competitive companion again, the old file will be opened."
+    },
+    {
+        "name": "Language",
+        "desc": "UI Language (need restart to apply)",
```

Comment:
I think we can use a message box to remind the user to restart, like when the editor theme changes under Windows dark theme?